test/topology/suite.yaml

```diff
@@ -5,3 +5,4 @@ topology:
     replication_factor: 3
 disable:
     - test_null
+    - test_cassandra_10250
```

Comment:
For the Cassandra repro, I added it here because using pytest-async and xfail gives an error:

    RuntimeError: cannot schedule new futures after shutdown

This will all go away once we enable raft in a subsequent pull request and the test is not expected to fail anymore.

Reply:
This "cannot schedule new futures after shutdown" thing looks like a bug in the test framework. A bug which you introduced in this series which introduced the futures thing, right? So let's fix it instead of hiding it. Your framework will need to be able to handle xfailing tests as well. Maybe in this case you can say let's wait a few hours or days and it will stop being xfail - but in other cases you may want to have an xfail for a long time and it should be ok.

Reply:
test_null was removed in v8 and we haven't seen this happen anymore.